V1_WhatCanYouBuild.md

```diff
@@ -6,15 +6,23 @@ is: 'published'
 
 ## What can you build?
 
-The Watson Work Services platform can support apps in Watson Workspace, or you can make them available on their own. That's great, but what's that mean? You can build apps that can:
+With the Watson Work Services platform, you can build apps that can:
```

Comment:
I think we need to keep one aspect of the old statement here that speaks to "apps in Workspace" as well as "apps that are available on their own (i.e., not workspace)"

Reply:
Since we say that on the `Get Started` page right before this I thought that felt a bit unnecessary here. 

> With Watson Work Services, you can build capabilities to customize your team's use of Watson Workspace, and create apps that will integrate with Watson Workspace for anyone to add to their spaces, and you can also develop apps for other business tools.

I don't think the string `With the WWS platform, you can build apps that can:` precludes the "outside of Workspace" scenario, just felt cleaner. Thoughts?

Reply:
I agree with the cleanliness, only thing is developers just don't read things "in sequence", so seeing "What can you build?" may draw their attention away - so redundancy in this case may be better.